PocketKit/Sources/PocketKit/Main/RegularMainCoordinator.swift

```diff
@@ -194,7 +200,6 @@ class RegularMainCoordinator: NSObject {
         readerSubscriptions = []
         model.home.selectedReadableViewModel = nil
         model.home.selectedSlateDetail?.selectedReadableViewModel = nil
-        model.myList.archivedItemsList.selectedReadable = nil
```

Comment:
@jacobsimeon this is the area where it fixes the issue on the iPad. Setting it nil was causing deselection when we still want the highlight.